pom.xml

```diff
@@ -192,10 +192,7 @@
 				  <licenseName>apache_v2</licenseName>
 				  <addJavaLicenseAfterPackage>false</addJavaLicenseAfterPackage>
 				  <excludes>
-					<exclude>**/font-awesome.css</exclude>
```

Comment:
Why is this resource excluded?

Reply:
It was already exluded. It is used in demo. It's a font-awesome file that should not be target by the licence plugin.

Reply:
> Why is this exclusion removed.
It wasn't 😂